corehq/apps/app_manager/add_ons.py

```diff
@@ -155,6 +155,10 @@ def _grandfathered(slug, app):
         name=_("New Case Lists Created Empty"),
         description=_("When adding a new case list, don't include a registration and followup form."),
     ),
+    "partial_commcare_translations": AddOn(
+        name=_("Support Partial CommCare Translations"),
```

Comment:
This is sometimes "CommCare" sometimes "UI" translations

![image](https://user-images.githubusercontent.com/1471773/53449115-1f7c9e00-39e7-11e9-8bdc-27bbb9e06b67.png)